<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblBeginDate">
     <property name="text">
      <string>С:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDateEdit" name="edtBegDate"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>По:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QDateEdit" name="edtEndDate"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblorgStruct">
     <property name="text">
      <string>Подразделение:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="COrgStructureComboBox" name="cmbOrgStruct"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="lblChiefResearcher">
     <property name="text">
      <string>Главный исследователь:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbResearcher"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblPartner">
     <property name="text">
      <string>Контрагент:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="COrgComboBox" name="cmbPartner"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="chkIsValid">
     <property name="text">
      <string>Только действующие</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QCheckBox" name="chkIsEnrollment">
     <property name="text">
      <string>Открыт набор пациентов</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>COrgComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
